### PR TITLE
[Hub Generated] Review request for Microsoft.Insights to add version stable/2018-03-01

### PR DESCRIPTION
**Pull request by sivanguetta**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/sivanguetta/azure-rest-api-specs/dev-monitor-Microsoft.Insights-2018-03-01-count-agg...master/).

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json [hunk 1]
```diff
@@ -797,7 +797,8 @@
             "Average",
             "Minimum",
             "Maximum",
-            "Total"
+            "Total",
+            "Count"
```
Comment (raych1) [inline specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json]: Adding new value in enum allowed values is a breaking change to SDK. Suggest to change this in new API version.

Comment (sivanguetta) [reply specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json]: @raych1 
Thanks for the review.

It is a bug fix. There is a misalignment between metric alerts RP and metric alerts swagger.
Currently Metric alerts RP returns all the 5 values while swagger documented only 4 values.

As the RP already supports the new property value, can we push it to the current API version?
Any other suggestion how to fix it?

Comment (ryansbenson) [reply specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json]: This enum is modeled as a string so it shouldn't affect the SDK. For the sake of this comment, someone from the SDK team should ack on this.

**Review by ryansbenson — COMMENTED** [review]
Approved from ARMs side

**Review by raych1 — APPROVED** [review]

File: specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json (hunk 1 above)
Comment (raych1) [reply specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json]: It's ok to align with RP behavior in this case.

**Pull request merged**